os/os-mac.h

```diff
@@ -117,3 +123,58 @@ static inline bool os_cpu_has(cpu_features feature)
 }
 
 #endif
+
+/*
+ * discard_pages should not be run within Rosetta. Native arm64 or x86 only. 
+ */
+
+#define MMAP_CHUNK_SIZE		(1024 * 1024 * 1024)
```

Comment:
Why is the chunk only 1GByte? Is it bad to map a larger amount like 4GBytes on a machine that might only have 8GBytes?